qgreenland/config/helpers/steps/decompress.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+from qgreenland.models.config.step import ConfigLayerCommandStep
+
+
+def decompress_step(
+    *,
+    input_file: str,
+    decompress_type: Optional[str] = 'unzip',
```

Comment:
I think we should change the type from `Optional[str]`; this implies that the value can be null, but it can only be `unzip` right now. I'd prefer the type `Literal['unzip']` for now (still with the default value), then we can expand on it later. I think we should keep the runtime check.